sklearn/metrics/tests/test_score_objects.py

```diff
@@ -47,7 +51,7 @@ class EstimatorWithoutFit(object):
     pass
 
 
-class EstimatorWithFit(object):
+class EstimatorWithFit(BaseEstimator):
```

Comment:
because ``cross_val_score`` uses ``set_params`` and ``clone``.
We have tests that check that ``cross_val_score`` works as long as ``get_params`` and ``set_params`` are present, right? --- Oh, looks like that is only for ``GridSearchCV``